src/reducers/ajaxStatusReducer.js

```diff
@@ -0,0 +1,16 @@
+import * as types from '../actions/actionTypes';
+import initialState from './initialState';
+
+function actionTypeEndsInSuccess(type) {
+  return type.substring(type.length - 8) == '_SUCCESS';
```

Comment:
Update to use triple equals. Does not change much, but you should start getting used to it.